dp/generate_climos.py

```diff
@@ -366,10 +370,29 @@ def update_metadata_and_time_var(input_file, t_start, t_end, climo_filepath):
             raise ValueError('Expected climo file to contain # time values in {}, but found {}'
                              .format(num_times_to_interval_set.keys(), cf.time_var.size))
 
+
```

Comment:
Also please add documentation for argument `operation`, including valid value set.